lib/gemstash/cli/info.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "gemstash"
+require "puma/control_cli"
+
+module Gemstash
+  class CLI
+    # This implements the command line setup task:
+    #  $ gemstash info
+    class Info < Gemstash::CLI::Base
+      include Gemstash::Env::Helper
+      def run
+        prepare
```

Comment:
Why we need prepare?

Reply:
It makes sure that if a user added some new manual changes configuration file, it should store it in the current env config variable before it tries to print it out!